<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="0" failed="0" skipped="10">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-16T17:12:02 IST" name="Failed suite [AjuSuite]" finished-at="2022-08-16T17:12:03 IST" duration-ms="471">
    <groups>
    </groups>
    <test started-at="2022-08-16T17:12:02 IST" name="ChromeTest(failed)" finished-at="2022-08-16T17:12:03 IST" duration-ms="186">
      <class name="testpack.VerifyHedersOfApp">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:02 IST" name="launchBrowser" finished-at="2022-08-16T17:12:02 IST" duration-ms="6" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="createPOMObject()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:02 IST" name="createPOMObject" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650122997" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPOMObject -->
        <test-method is-config="true" signature="loginPageToApplication()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="loginPageToApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123018" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageToApplication -->
        <test-method signature="verifyReportButton()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="verifyReportButton" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyReportButton -->
        <test-method is-config="true" signature="logoutFromApplication()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="logoutFromApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123030" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="loginPageToApplication()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="loginPageToApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123036" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageToApplication -->
        <test-method signature="verifyTaskButton()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="verifyTaskButton" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTaskButton -->
        <test-method is-config="true" signature="logoutFromApplication()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="logoutFromApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123043" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="clearObject()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="clearObject" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123046" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearObject -->
        <test-method is-config="true" signature="closedBrowser()[pri:0, instance:testpack.VerifyHedersOfApp@635eaaf1]" started-at="2022-08-16T17:12:03 IST" name="closedBrowser" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123106" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closedBrowser -->
      </class> <!-- testpack.VerifyHedersOfApp -->
      <class name="testpack.TestClass1">
        <test-method is-config="true" signature="beforetest()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:02 IST" name="beforetest" finished-at="2022-08-16T17:12:02 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforetest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="beforeClass" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123057" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="beforeMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123060" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testB()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="testB" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="afterMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123069" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="beforeMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123077" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testF()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="testF" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testF -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="afterMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123085" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="beforeMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123088" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testA()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="testA" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="afterMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123096" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="AfterClass()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="AfterClass" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123098" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterClass -->
        <test-method is-config="true" signature="aftertest()[pri:0, instance:testpack.TestClass1@1817d444]" started-at="2022-08-16T17:12:03 IST" name="aftertest" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123102" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- testpack.TestClass1 -->
    </test> <!-- ChromeTest(failed) -->
    <test started-at="2022-08-16T17:12:03 IST" name="firefoxTest(failed)" finished-at="2022-08-16T17:12:03 IST" duration-ms="142">
      <class name="testpack.VerifyHedersOfApp">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="launchBrowser" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="createPOMObject()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="createPOMObject" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123267" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPOMObject -->
        <test-method is-config="true" signature="loginPageToApplication()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="loginPageToApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123270" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageToApplication -->
        <test-method signature="verifyReportButton()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="verifyReportButton" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyReportButton -->
        <test-method is-config="true" signature="logoutFromApplication()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="logoutFromApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123289" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="loginPageToApplication()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="loginPageToApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123293" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageToApplication -->
        <test-method signature="verifyTaskButton()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="verifyTaskButton" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTaskButton -->
        <test-method is-config="true" signature="logoutFromApplication()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="logoutFromApplication" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123304" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="clearObject()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="clearObject" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123308" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearObject -->
        <test-method is-config="true" signature="closedBrowser()[pri:0, instance:testpack.VerifyHedersOfApp@7334aada]" started-at="2022-08-16T17:12:03 IST" name="closedBrowser" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123395" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closedBrowser -->
      </class> <!-- testpack.VerifyHedersOfApp -->
      <class name="testpack.TestClass1">
        <test-method is-config="true" signature="beforetest()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="beforetest" finished-at="2022-08-16T17:12:03 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforetest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="beforeClass" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123314" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="beforeMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123325" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testB()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="testB" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="afterMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123365" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="beforeMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123369" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testF()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="testF" finished-at="2022-08-16T17:12:03 IST" duration-ms="1" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testF -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="afterMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123375" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="beforeMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123379" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testA()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="testA" finished-at="2022-08-16T17:12:03 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field
Base cannot be resolved or is not a field

at testpack.VerifyHedersOfApp.launchBrowser(VerifyHedersOfApp.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="afterMethod" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123386" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="AfterClass()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="AfterClass" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123390" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterClass -->
        <test-method is-config="true" signature="aftertest()[pri:0, instance:testpack.TestClass1@42e99e4a]" started-at="2022-08-16T17:12:03 IST" name="aftertest" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1660650123393" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- testpack.TestClass1 -->
    </test> <!-- firefoxTest(failed) -->
  </suite> <!-- Failed suite [AjuSuite] -->
</testng-results>
